condDatasetSubmitter.py

```diff
@@ -423,6 +435,9 @@ def createCMSSWConfigs(options,confCondDictionary,allRunsAndBlocks):
             driver_command += '--customise %s ' % (details['customise'])
         if details['era'] != "":
             driver_command += "--era %s " % (details['era'])
```

Comment:
@franzoni  Here are the contents of the details dictionary
{'eventcontent': 'RECO,DQM', 'dumppython': False, 'inputcommands': '', 'reqtype': 'PR', 'magfield': '', 'inclparents': 'False', 'customise': '', 'runUnscheduled': '', 'output': '', 'datatier': 'RECO,DQMIO ', 'custcommands': '', 'procname': 'reRECO', 'steps': 'RAW2DIGI,L1Reco,RECO,EI,PAT,DQM', 'era': 'Run2_2017', 'custconditions': ''}
and the cmsDriver command to run is 
cmsDriver.py PR -s RAW2DIGI,L1Reco,RECO,EI,PAT,DQM --processName reRECO --data --scenario pp --datatier RECO,DQMIO  --conditions 92X_dataRun2_PROMPT_ECalPulseShapes_w21_2017 --python_filename NEWCONDITIONS0.py --no_exec -n 100 --eventcontent RECO,DQM --era Run2_2017 --runUnscheduled 
all the subsequent checks are in agreement.